src/riak_kv_compile_tab.erl

```diff
@@ -211,15 +211,11 @@ get_table_ddl(_Table) ->
     DDL = {}, %% not used by caller
     {ok, Module, DDL}.
 
-is_table_supported(DDLRecCap, Table = <<"my_type2">>) ->
-    is_table_supported_not_active(DDLRecCap, Table);
-is_table_supported(DDLRecCap, Table) ->
-    is_table_supported_active(DDLRecCap, Table).
-
-is_table_supported_active(_DDLRecCap, _Table) ->
```

Comment:
Yes, these functions essentially returned constants for testing and were only used in that context.